airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -776,6 +777,12 @@ def run_load(self,
         :param quote_character: The value that is used to quote data sections in a CSV
             file.
         :type quote_character: string
+        :param ignore_unknown_values: [Optional] Indicates if BigQuery should allow
+            extra values that are not represented in the table schema.
+            If true, the extra values are ignored. If false, records with extra columns
```

Comment:
nit: text formatting

Reply:
@fenglu-g This if for Sphinx documentation.

Reply:
gotcha.